src/main/resources/public/ts/controllers/controller.ts

```diff
@@ -1023,7 +1023,8 @@ export const calendarController =  ng.controller('CalendarController',
                 }
             });
         $scope.calendarAsContribRight = unique($scope.calendarAsContribRight);
-        $scope.selectedCalendarInEvent.push($scope.calendarAsContribRight[0]);
+        let defaultCalendar : Calendar = $scope.calendarAsContribRight.find(cal => cal.is_default == true);
+        $scope.selectedCalendarInEvent.push(defaultCalendar ? defaultCalendar : $scope.calendarAsContribRight[0]);
```

Comment:
On est sur que calendrAsContrib sera toujours défini ? Tu risques de renvoyer un "undefined" dans ton tableau selected ou cannot read property of undefined/null